frontend/src/components/email-button.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Dropdown, DropdownButton } from "react-bootstrap";
+
+export class EmailButton extends React.Component {
+    static propTypes = {
+        data: PropTypes.arrayOf(PropTypes.number).isRequired
+    };
+
+    // email applicants
+    email = () => {
+        console.log(this.props.data);
+        let emails = [...this.props.data];
+
+        var a = document.createElement("a");
+        a.href =
+            emails.length === 1
+                ? "mailto:" + emails[0] // if there is only a single recipient, send normally
+                : "mailto:?bcc=" + emails.join(","); // if there are multiple recipients, bcc all
+        a.click();
```

Comment:
I'm not happy with this. You're also polluting the DOM with an `a` that never gets destroyed after it's created. There should be another way. At a minimum, make a persistent `a` whose `href` gets updated.